chapters/methodology.tex

```diff
@@ -11,6 +11,8 @@ \subsection{Architecture}
 \subsection{State Machine}
 Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
 
-\subsection{User Interface}
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
+\subsection{User Interface} \label{sec:user_interface}

```

Comment:
As mentioned before in #1 , more line breaks instead of word wrapping would be beneficial for commenting 😉 
However this is not blocking this PR...

Reply:
Should be resolved now!